scripts/install_stacks.sh

```diff
@@ -19,7 +19,7 @@ git clone --depth 1 --branch master https://github.com/stacks-network/stacks-blo
 echo "[ install_stacks.sh ] - Build and install stacks-blockchain binary -> /usr/local/bin/stacks-node"
 cargo build --features monitoring_prom,slog_json --release --bin stacks-node
 sudo cp -a "${HOME}/stacks-blockchain/target/release/stacks-node" "/usr/local/bin/stacks-node"
-
+sudo rm -rf "${HOME}/stacks-blockchain"
```

Comment:
```suggestion
```

i would keep the source code on the system for rebuilds

Reply:
Understood, will remove the remove